static/sass/styles.scss

```diff
@@ -8,6 +8,9 @@ $color-tabs-active-bar: #e95420;
 $color-brand: #7c355d;
 $color-dark: #262626;
 $color-paper: #f3f3f3;
+
+// XXX: remove once this is merged: https://github.com/canonical/vanilla-framework/pull/4810
```

Comment:
The convention for `XXX` comments should be `
// XXX: {date} by {author} - {reason for the change}
// {Link to issue}
`

Reply:
removed,  pr's already in.